src/editor-spacing.scss

```diff
@@ -1,5 +1,6 @@
-:root, :host {
+:root,
+:host {
   --spacing-half: calc(0.5rem * var(--spacing-multiplier));
   --spacing-eight: calc(var(--spacing-1) / 1.25);
   --spacing-four: calc(var(--spacing-eight) / 2);
-}
+}
```

Comment:
Also highly picky but do you mind adding the blank lines back in at the end of files? There are a few examples of this

Reply:
Had formatting on save setup from innovation day project! (Whooops) - Done in [01b0d33](https://github.com/RaspberryPiFoundation/editor-ui/pull/314/commits/01b0d33f3169428ab44f471d9a8f8abffa5938fd)